src/NUnitEngine/nunit.engine/Services/AgentRecord.cs

```diff
@@ -29,14 +29,12 @@ namespace NUnit.Engine.Services
 {
     internal sealed class AgentRecord
     {
-        private AgentRecord(Guid id, Process process, ITestAgent agent)
+        private AgentRecord(Process process, ITestAgent agent)
```

Comment:
Yeah... originally, I think the idea I had was that the agency might start a process but some other, second-level process might actually register. That turned out never to happen. Good lesson in YAGNI.